common/src/main/java/bio/terra/catalog/service/dataset/DatasetDao.java

```diff
@@ -37,6 +41,17 @@ public DatasetDao(NamedParameterJdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
+  @Autowired ObjectMapper objectMapper;
+
+  public ObjectNode toJsonNode(String json) {
```

Comment:
I guess there's no way to test that code otherwise, since we have no way to put invalid JSON into postgres JSONB. It's a shame the code is needed at all.

In that case I would make it package protected (remove `public`) and add `@VisibleForTesting`.